.changeset/few-toes-drive.md

```diff
@@ -0,0 +1,9 @@
+---
+"@ledgerhq/types-live": patch
```

Comment:
Now we need to use `minor` instead of `patch`

Reply:
https://github.com/LedgerHQ/ledger-live/pull/8375/commits/6d9abec7eb75f83c9695bbaca9c61a7fc592ac38